src/OmniSharp.Abstractions/FileWatching/IFileSystemWatcher.cs

```diff
@@ -9,8 +9,8 @@ public interface IFileSystemWatcher
         /// <summary>
         /// Call to watch a file or directory path for changes.
         /// </summary>
-        /// <param name="fileOrDirectoryPath">The file or directory path to watch.</param>
+        /// <param name="pathOrExtension">The file or directory path to watch.</param>
         /// <param name="callback">The callback that will be invoked when a change occurs in the watched file or directory.</param>
-        void Watch(string fileOrDirectoryPath, FileSystemNotificationCallback callback);
+        void Watch(string pathOrExtension, FileSystemNotificationCallback callback);
```

Comment:
See question at https://github.com/OmniSharp/omnisharp-roslyn/pull/1053#discussion_r156129196

Reply:
At the least, there should be a corresponding update to the XML doc comment, since it no longer seems to match the parameter name.

Reply:
@DustinCampbell, ah missed the doc. Will update in the original PR #1053